our-process/policies.md

```diff
@@ -110,9 +122,8 @@ of individual maintainers. Buggy, unmaintained software may be removed from our
 any time. We also ask maintainers that they get in touch with us if they do need
 to step down from maintaining a tool.
 
+### Requesting package removal from the pyOpenSci ecosystem
 
-
-### Requesting Package Removal
 In the unlikely scenario that a contributor of a package requests removal of their
-package from the suite, we retain the right to maintain a version of the package in our
-suite for archival purposes.
+package from our ecosystem, we retain the right to archive a version of the package in our
```

Comment:
```suggestion
package from our ecosystem, we retain the right to offer the currently released version of the package in our
```

Reply:
@willingc question about this fix. i like the language but wonder - does this suggest that we are taking on maintenance of the package in this case? i guess i worry about that because we don't have the capacity to maintain new packages like rOpenSci does. But we might want to create an archive of the repository for safe keeping in the form of a fork, etc ? thoughts?

Reply:
@lwasser I think it makes sense to be explicit about what we are doing. I don't think it is wise to imply that we will maintain any project. I was thinking along the lines of making a copy of the current release at the time of removal.

Reply:
ahhh ok . How about language like:
***
> we retain the right to provide an archived copy of the package, in it's most recently release state the currently released version of the package in our ... 

:::{important}
pyOpenSci will never take on maintenance responsibilities of a community created package. But we will create archival copies of packages and support finding new maintainers for existing unmaintained packages as it makes sense. 
:::

***

Reply:
i took this out of context - 

it now reads

> In the unlikely scenario that a contributor of a package requests removal of their
> package from our ecosystem, we retain the right offer the last / most recently released version of that package in our ecosystem for archival purposes only.

i forgot it said for archival purposes only and the way the gui presents things it only shows what has changed! this is fixed locally now